### PR TITLE
Headless report

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?

* Ticket: #
* Funded by: DFAT
* Description: 
  * fix unevaluated expressions in inasafe realtime
  * fix duplicated impact layer on the map report for multi-exposure

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/gis/generic_expressions.py [hunk 1]
```diff
@@ -287,14 +295,21 @@ def beautify_time(inasafe_time, feature, parent):
     """
     _ = feature, parent  # NOQA
     try:
-        datetime_object = datetime.datetime.strptime(
-            inasafe_time, '%Y-%m-%dT%H:%M:%S.%f')
-    except ValueError:
+        gmt_offset_str = inasafe_time[-6:-4]
```
Comment (ismailsunni) [inline safe/gis/generic_expressions.py]: It seems duplicate code with the one in beautify_date. Can you make it as a function?

File: safe/gis/generic_expressions.py [hunk 2]
```diff
@@ -252,14 +253,21 @@ def beautify_date(inasafe_time, feature, parent):
...
```
Comment (Gustry) [inline safe/gis/generic_expressions.py]: This looks dangerous. Is there not a function in datetime module to get the offset?

Comment (ismailsunni) [reply safe/gis/generic_expressions.py]: Agree with @Gustry , perhaps you can check this one https://stackoverflow.com/questions/12281975/convert-timestamps-with-offset-to-datetime-obj-using-strptime

Comment (myarjunar) [reply safe/gis/generic_expressions.py]: I did a lot of googling for this and not sure there is such a function in datetime library.

File: safe/gis/generic_expressions.py (hunk 1 above)
Comment (Gustry) [reply safe/gis/generic_expressions.py]: +1

File: safe/gis/generic_expressions.py (hunk 2 above)
Comment (Gustry) [reply safe/gis/generic_expressions.py]: Ok, strange from a famous python module.
So make it safer as possible and future proof ;-)

Can you move the logic to safe.utilities folder in a single function (linked to the comment from Ismail) and add some quick tests?

Comment (myarjunar) [reply safe/gis/generic_expressions.py]: thanks @ismailsunni @Gustry , ok so with `parse` method we could generally convert it into the datetime object. but my intention was for knowing how much the time difference is so we could convert it to localtime. we could figure it put later

**Pull request merged**